### PR TITLE
enhance: Enable alter database props in rootcoord

### PR DESCRIPTION
**Pull request by weiliu1031**
issue: #30040

### CODE REVIEW

File: internal/rootcoord/alter_database_task.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		return fmt.Errorf("alter collection failed, collection name does not exists")
+	}
+
+	return nil
+}
+
+func (a *alterDatabaseTask) Execute(ctx context.Context) error {
+	// Now we only support alter properties of collection
+	if a.Req.GetProperties() == nil {
+		return errors.New("only support alter collection properties, but collection properties is empty")
```
Comment (bigsheeper) [inline internal/rootcoord/alter_database_task.go:47]: database properties, please modify the comment as well

File: internal/rootcoord/alter_database_task.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+	oldDB, err := a.core.meta.GetDatabaseByName(ctx, a.Req.GetDbName(), a.ts)
+	if err != nil {
+		log.Warn("get database failed during changing database props",
+			zap.String("databaseName", a.Req.GetDbName()), zap.Uint64("ts", a.ts))
+		return err
+	}
+
+	newDB := oldDB.Clone()
+	updateProperties(oldDB.Properties, a.Req.GetProperties())
```
Comment (bigsheeper) [inline internal/rootcoord/alter_database_task.go:58]: should assign return value to the newDB?

File: internal/rootcoord/root_coord.go [hunk 3]
```diff
@@ -1290,6 +1290,58 @@ func (c *Core) AlterCollection(ctx context.Context, in *milvuspb.AlterCollection
...
+
+		metrics.RootCoordDDLReqCounter.WithLabelValues(method, metrics.FailLabel).Inc()
+		return merr.Status(err), nil
+	}
+
+	metrics.RootCoordDDLReqCounter.WithLabelValues(method, metrics.SuccessLabel).Inc()
+	metrics.RootCoordDDLReqLatency.WithLabelValues(method).Observe(float64(tr.ElapseSpan().Milliseconds()))
+	metrics.RootCoordDDLReqLatencyInQueue.WithLabelValues(method).Observe(float64(t.queueDur.Milliseconds()))
+
+	log.Info("done to alter database",
```
Comment (bigsheeper) [inline internal/rootcoord/root_coord.go:1338]: log.Ctx

**Review by longjiquan — COMMENTED** [review]
LGTM generally.
Quick question: should we expose this api to milvus service (milvus.proto)?

**Comment by weiliu1031** [comment]
> LGTM generally. Quick question: should we expose this api to milvus service (milvus.proto)?

will expose it  in next pr

**Pull request merged**